sys-libs/timezone-data/timezone-data-2021a.ebuild

```diff
@@ -150,28 +150,29 @@ configure_tz_data() {
 	fi
```

Comment:
For existing users of a symlink and `/etc/timezone`:

```
	if [[ -L "${etc_lt}" ]] ; then
		einfo "Assuming your ${etc_lt} symlink is what you want; skipping update."
		return 0
	fi
```

Should we suggest users to remove `/etc/localtime` manually (or do it for them)?

Reply:
I added an ewarn to the ebuild with instructions.

Reply:
Oh, I guess you meant when /etc/timezone is a symlink and /etc/localtime is a symlink already.

I guess my thought process is that once /etc/localtime is a symlink, we treat it as the authority and ignore /etc/timezone from that point forward. Eventually, we remove references to /etc/timezone from documentation and just have users create the /etc/localtime symlink themselves (or using `timedatectl set-timezone` from systemd).